onnxruntime/core/session/compile_api.cc

```diff
@@ -0,0 +1,242 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if !defined(ORT_MINIMAL_BUILD)
+
+#include "core/session/compile_api.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "core/framework/error_code_helper.h"
+#include "core/session/abi_session_options_impl.h"
+#include "core/session/inference_session.h"
+#include "core/session/model_compilation_options.h"
+#include "core/session/onnxruntime_session_options_config_keys.h"
+#include "core/session/ort_apis.h"
+#include "core/session/ort_env.h"
+#include "core/session/utils.h"
+
+#define ORT_C_API_RETURN_IF_ERROR(expr)                 \
```

Comment:
could ORT_API_RETURN_IF_STATUS_NOT_OK in error_code_helper.h be used?

Reply:
A yes, thanks.

Reply:
Removed.